neotron-bmc-commands/src/lib.rs

```diff
@@ -137,4 +137,24 @@ pub enum Command {
 	/// * Length: 4
 	/// * Mode: R/W
 	I2cBaudRate = 0x64,
+	/// # Speaker Duration
```

Comment:
Please add these to the README.md. Or, even better, delete the chunk of text describing the registers in the README and point to a new `neotron-bmc-commands/README.md` Then in that README, just point people at this list. Otherwise there's a risk they'll get out of sync in future (and I think they're actually out of sync already).